changelog/CHANGELOG.tmpl

```diff
@@ -1,7 +1,13 @@
+# Table of Contents
+
+{{ range . -}}
+  * [Changelog for {{ .Version }}](#changelog-for-owncloud-desktop-client-{{ .Version | replace "." ""}}-{{ .Date | lower -}})
+{{ end -}}
+  * [Changelog for 2.6.1 versions and below](#changelog-for-261-versions-and-below)
 {{ $allVersions := . }}
 {{- range $index, $changes := . }}{{ with $changes -}}
-Changelog for ownCloud Desktop Client [{{ .Version }}] ({{ .Date }})
-=======================================
+# Changelog for ownCloud Desktop Client [{{ .Version }}] ({{ .Date }})
```

Comment:
You should add one or two blank lines above this heading. Right now, rendering may or may not work. Also, the resulting generated code is kind of hard to read.

```suggestion


# Changelog for ownCloud Desktop Client [{{ .Version }}] ({{ .Date }})
```

Reply:
I can tell that we have that in all repos where we use this TOC generating code the same and it works without any issues. The code was created by mbarz and I have tested it successfully locally using calens. I would therefore keep that as it is to have that harmonized with the other templates.

Reply:
Should be taken into account upon the next update then, I guess. The generated code is still pretty ugly.